maintainers.txt

```diff
@@ -1,8 +1,6 @@
 p.hooper@elifesciences.org
 h.drury@elifesciences.org
 c.donkin@elifesciences.org
-jennifer.spencer@yld.io
-hamant@yld.io
-javier.yenes@yld.io
-semra.shukrieva@yld.io
-#xpub-tech
```

Comment:
removing the Slack channel?

Reply:
Ah I thought this was a rougue comment - oops

Reply:
Could the format be made a bit clearer here, maybe `elifesciences.slack.com#xpub-tech`?

Reply:
Ah yes - does that format work too? Does look clearer :)

Reply:
It doesn't work, but it can be made to work if everyone agrees that should be the format

Reply:
I can't imagine there won't be another person confused by the leading `#` ... this is what confused me. Although once you know its for slack it makes sense. I like @thewilkybarkid suggestion as that actually works in the browser too :)

Reply:
https://github.com/elifesciences/elife-jenkins-workflow-libs/pull/16